docs/index.rst

```diff
@@ -74,6 +74,7 @@ or ``theme.conf`` for more details.
     admonitions
     content_tabs
     mermaid_diagrams
+    graphviz
```

Comment:
I think it would less cluttered if we group these docs together under a "Graphs" `toctree`.

Reply:
To be clear, I think the font issue also exists on Linux and macOS --- it isn't customary to have the Google fonts available as system fonts.  I think it is just that the fontconfig library always picks some font if the requested font can't be found and doesn't warn about it.  In contrast, on Windows graphviz does warn, and we are treating warnings as errors in this theme's docs.

Note that for svg rendering it is kind of subtle --- at build time the font is only used to determine the size of the label.  The svg file will just specify whichever font name was requested, even if it is not available at build time.  So as long as the font is available to the browser, the labels will be displayed with the correct font.  But the layout may have been computed with the wrong sizes, which in most cases may not be very noticable.

Reply:
> I think it would less cluttered if we group these docs together under a "Graphs" `toctree`.

Done.